packages/ethereum-contracts/contracts/superfluid/Superfluid.sol

```diff
@@ -499,6 +501,7 @@ contract Superfluid is
             }
         } else {
             newCtx = ctx;
+            // no change to the stamped ctx, no stampin gneeded
```

Comment:
*small typo
fix: no stamping needed

Reply:
it was not necessary comment, something added while testing out some logic. It is deleted for now.

Reply:
> @hellwolf did you still want to add the bad agreement case?

yea, gonna add it